package.json

```diff
@@ -36,6 +39,9 @@
     "@tsconfig/docusaurus": "1.0.5",
     "eslint": "8.14.0",
     "eslint-plugin-mdx": "1.17.0",
+    "husky": ">=6",
+    "lint-staged": ">=10",
+    "prettier": "^2.6.2",
```

Comment:
can you pin these? :)

Reply:
Do you know if they will be pinned automatically by renovate later?

Reply:
I believe so, depending on which default config renovate uses. But either way I think we want to have the same behaviour throughout core/CLI/website

Reply:
It will but it prevents an unnecessary PR :)
(don't forget to pin the other two)

Reply:
fixed